File: geekbang/algorithm/source-code/1/lc26 删除有序数组中的重复项.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = 0
        for i in range(0, len(nums)):
            if i == 0 or nums[i] != nums[i - 1]:
                nums[n] = nums[i]
                n += 1
        return n

File: geekbang/algorithm/source-code/1/lc26 删除有序数组中的重复项.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let n = 0;
    for (let i = 0; i < nums.length; i++) {
        if (i == 0 || nums[i] != nums[i - 1]) {
            nums[n] = nums[i];
            n++;
        }
    }
    return n;
};